generator/integration_tests/BUILD.bazel

```diff
@@ -59,6 +59,70 @@ cc_grpc_library(
     name = "google_cloud_cpp_generator_testing_cc_grpc",
     srcs = [":google_cloud_cpp_generator_testing_proto"],
     grpc_only = True,
-    visibility = [":__subpackages__"],
     deps = [":google_cloud_cpp_generator_testing_cc_proto"],
 )
+
+filegroup(
```

Comment:
TIL. `filegroup`. It looks tidier than what we currently do with: `SOURCE_GLOB = "**/*.cc"` in our generated `BUILD.bazel`s. I do not think it is worth updating though.

Reply:
Ack